engine/render/src/render/render_script_camera.cpp

```diff
@@ -246,6 +260,7 @@ namespace dmRender
         // READ-ONLY
         {"get_projection",          RenderScriptCamera_GetProjection},
         {"get_view",                RenderScriptCamera_GetView},
+        {"get_enabled",             RenderScriptCamera_GetEnabled},
```

Comment:
This is added because we don't have a way to check this from render scripts otherwise. We could consider adding a "set_enabled" as well, but it's not needed for now